cmd/wsm/wsm.go

```diff
@@ -222,7 +263,7 @@ func DeployCmd() *cobra.Command {
 				os.Exit(1)
 			}
 
-			wb := crd.NewWeightsAndBiases("charts/"+helm.WandbChart, vals)
+			wb := crd.NewWeightsAndBiases(chartPath, vals)
```

Comment:
This we have to make wb specific, but I wonder if we could think about a way to abstract this away